### PR TITLE
chore(gen_vimdoc): correct minimum python version

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: scripts/gen_vimdoc.py [hunk 1]
```diff
@@ -96,7 +96,6 @@
             'win_config.c',
             'tabpage.c',
             'ui.c',
-            'extmark.c',
```
Comment (jamessan) [inline scripts/gen_vimdoc.py:99]: Why is this removed?

Comment (zeertzjq) [reply scripts/gen_vimdoc.py:99]: There seems to be already a same entry above.

Comment (dundargoc) [reply scripts/gen_vimdoc.py:99]: Zeer is correct.

I guess I could have put the fix in the typo squash PR or in a separate PR altogether but felt like it was too small a change for that.

Comment (dundargoc) [reply scripts/gen_vimdoc.py:99]: I'll remove that change and create a new PR for that.

**Comment by dundargoc** [comment]
Shall I just add this to the typo PR or something?

**Comment by jamessan** [comment]
Are you applying this change elsewhere?  If not, why is the PR closed? :)

**Comment by dundargoc** [comment]
I felt it was too insignificant/low effort to be worth it.

**Comment by clason** [comment]
Why? We merge PRs for stupid typos -- this strikes me as much more significant.

**Comment by dundargoc** [comment]
OK, I guess I misjudged its worth then. My bad.

**Comment by clason** [comment]
But I think we're bumping the minimum version to 3.7 throughout the project since 3.6 also reached end-of-life last year.

**Comment by dundargoc** [comment]
Does that apply for our internal tools as well? This wasn't a "python 3.5 is deprecated" but a "this script will not work with python 3.5". Shall I change to 3.7 them?

**Comment by clason** [comment]
I'll defer to @jamessan on that -- he knows best whether bumping that would exclude some currently supported platforms.
